addons/events/XEH_postInit.sqf

```diff
@@ -11,3 +11,18 @@
         };
     } forEach allVariables GVAR(eventNamespaceJIP);
 }, []] call CBA_fnc_execNextFrame;
+
+
+if (isServer) then {
+    CBA_clientID = [0, 2] select isMultiplayer;
+    addMissionEventHandler ["PlayerConnected", {
+        params ["_id", "_uid", "_name", "_jip", "_owner"];
+        TRACE_5("PlayerConnected eh",_id,_uid,_name,_jip,_owner);
+
+        if (_owner != 2) then {
+            CBA_clientID = _owner;
```

Comment:
Are `PlayerConnected` handlers executed in scheduled or unscheduled space? (If scheduled, I see a race condition with the `CBA_clientID` variable here)